StackOverflow/Controller/QuestionsCell.h

```diff
@@ -0,0 +1,18 @@
+//
+//  QuestionsCell.h
+//  StackOverflow
+//
+//  Created by Mali on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface QuestionsCell : UITableViewCell
+
+@property (weak, nonatomic) IBOutlet UIImageView *acceptedAnswerImage;
+@property (weak, nonatomic) IBOutlet UILabel *questionTitle;
+@property (weak, nonatomic) IBOutlet UILabel *tags;
+@property (weak, nonatomic) IBOutlet UILabel *answerCount;
```

Comment:
answerCountLabel. Remember the Swift documentation about names being meaningful at the point of use. If I look at this I think it contains an int or a float value because it's a count, but actually it's not, it's a label. You MUST get your naming correct. If PRs have bad naming I will reject them.

Reply:
I will correct this thank you